pkg/kubelet/dockertools/docker.go

```diff
@@ -63,9 +64,9 @@ type DockerInterface interface {
 	StartContainer(id string, hostConfig *docker.HostConfig) error
 	StopContainer(id string, timeout uint) error
 	RemoveContainer(opts docker.RemoveContainerOptions) error
-	InspectImage(image string) (*docker.Image, error)
-	ListImages(opts docker.ListImagesOptions) ([]docker.APIImages, error)
-	PullImage(opts docker.PullImageOptions, auth docker.AuthConfiguration) error
+	InspectImage(image string) (*dockertypes.ImageInspect, error)
+	ListImages(opts dockertypes.ImageListOptions) ([]dockertypes.Image, error)
+	PullImage(opts dockertypes.ImagePullOptions, auth dockertypes.AuthConfig) error
```

Comment:
Change the function to conform to docker/engine-api#162.
See `RemoveContainer` in #23699.